source/installation-guide/installing-wazuh-server/sources_installation.rst

```diff
@@ -26,11 +24,40 @@ Installing Wazuh manager
 
       # apt-get install python gcc make libc6-dev curl policycoreutils automake autoconf libtool
 
+    These are the basic dependencies to build Wazuh's binaries if you are going to install them in the default directory but, if you want to install in any other directory, you need to install some new dependencies in order to build the CPython interpreter included in the Wazuh manager installation since v3.9.0. By default, ``make deps`` will download a pre-compiled version of CPython, built to be installed in ``/var/ossec``, otherwise, it will download a modified version of CPython sources. This sources will build CPython for the target directory and link the interpreter and the internal dependencies against the ``libwazuhext``.
```

Comment:
It could be simpler, something like:
"Install the following dependencies **only if the installation directory is not /var/ossec**. This is due to ``make deps`` will download a pre-compiled version of CPython (since v3.9.0), built to be installed in ``/var/ossec``. Otherwise, it will download a modified version of CPython sources and it will be necessary to compile it."

Reply:
Got it!